test/testenv/cmutil.go

```diff
@@ -0,0 +1,60 @@
+package testenv
+
+import (
+	"encoding/json"
+	"fmt"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+)
+
+// ClusterMasterHealthResponse is a representation of the health response by a Splunk cluster-master
+// Endpoint: /services/cluster/master/health
+type ClusterMasterHealthResponse struct {
```

Comment:
do we actually need this?I don't see multiple entries in the CM health response. Maybe just rename `ClusterMasterHealthEntry` to `ClusterMasterHealthResponse` and use it directly like `restResponse.Content.SearchFactorMet` ?

Reply:
@gaurav-splunk I kept the hierarchical order of API return for simplicity. In future I am thinking of adding more verifications in future therefore filtering more fields.